### PR TITLE
Bundler: use path block for modules

### PR DESCRIPTION
**Pull request by ericboehs**
Use the path directive in a ruby block https://bundler.io/man/gemfile.5.html#PATH

### CODE REVIEW

**Comment by erluti** [comment]
I'm a huge fan
👍

**Review by erluti — APPROVED** [review]

File: Gemfile.lock [hunk 1]
```diff
@@ -73,82 +73,34 @@ GIT
...
-  remote: modules/vba_documents
-  specs:
     vba_documents (0.0.1)
       aws-sdk-s3 (~> 1)
       sidekiq
-
-PATH
-  remote: modules/veteran_confirmation
-  specs:
+    veteran (0.0.1)
```
Comment (LindseySaari) [inline Gemfile.lock:100]: I'm thinking this was moved for alphabetizing purposes? Correct?

**Review by LindseySaari — APPROVED** [review]
💯

File: Gemfile.lock (hunk 1 above)
Comment (ericboehs) [reply Gemfile.lock:100]: Yeah. The diff looked a little weird here but this was all bundlers doing /shrug

**Pull request merged**